tiledb/sm/query/query.cc

```diff
@@ -328,12 +328,18 @@ Status Query::capnp(rest::capnp::Query::Builder* queryBuilder) const {
 
   std::vector<std::string> attributes = this->attributes();
 
-  rest::capnp::Map<::capnp::Text, rest::capnp::AttributeBuffer>::Builder
-      attributeBuffersBuilder = queryBuilder->initBuffers();
-  auto buffers = attributeBuffersBuilder.initEntries(attributes.size());
+  ::capnp::List<::tiledb::rest::capnp::AttributeBufferHeader>::Builder
+      attributeBufferHeaderBuilder =
+          queryBuilder->initBuffers(attributes.size());
+
+  uint64_t totalNumOfBytesInBuffers{};
```

Comment:
Can you change this to an explicit initialization to 0 here instead of using `{}`? E.g. `uint64_t totalNumOfBytesInBuffers = 0;`